From 79b58e8a0c4155134bdf680899fab22a7c01f35a Mon Sep 17 00:00:00 2001
From: Daniel Golle <daniel@makrotopia.org>
Date: Sat, 9 Mar 2024 00:46:18 +0000
Subject: [PATCH] DHTU: ship dhtu.conf
To: gnunet-developers@gnu.org

Without dhtu.conf GNUnet will create infinite loglines
ERROR No DHT underlays configured!

Ship the new config file to fix that.
---
 src/service/dht/Makefile.am | 3 +++
 src/service/dht/meson.build | 6 ++++++
 2 files changed, 9 insertions(+)

--- a/src/service/dht/Makefile.am
+++ b/src/service/dht/Makefile.am
@@ -10,6 +10,9 @@ libexecdir= $(pkglibdir)/libexec/
 pkgcfg_DATA = \
   dht.conf
 
+dist_pkgcfg_DATA = \
+  dhtu.conf
+
 if USE_COVERAGE
   AM_CFLAGS = --coverage -O0
   XLIB = -lgcov
--- /dev/null
+++ b/src/service/dht/dhtu.conf
@@ -0,0 +1,7 @@
+[dhtu-gnunet]
+ENABLED = YES
+
+[dhtu-ip]
+ENABLED = NO
+NSE = 4
+UDP_PORT = 6666
